### ISSUE TITLE
add "box" and "inline-box" without vendor perfixed of "display"

### ISSUE THREAD

**Issue by BenzLeung** [CSSLint/parser-lib#106]
Found "-webkit-box" and "-moz-box", but no "box". The same to "inline-box".

**Comment by kkirsche** [comment]
`display: box;` is a version of 2009.
`display: flexbox;` is a version of 2011.
`display: flex;` is the actual version.

**Comment by malept** [comment]
Yes, that's why I asked the question. Since it's part of the 2009 version of the flexbox spec, I couldn't find documentation of any browsers which actually implemented the non-prefixed values.

**Comment by kkirsche** [comment]
Paul Irish said: The old standard (Flexbox 2009) was implemented in Chrome since v4, Firefox since v2, and IE10 beta.

I'd recommend starting with [this StackOverflow](http://stackoverflow.com/questions/16280040/css3-flexbox-display-box-vs-flexbox-vs-flex) and going through it's links, etc.

**Comment by malept** [comment]
If you look at the [caniuse.com entry for flex-box](http://caniuse.com/#feat=flexbox), it indicates that in Chrome from versions 4 through 21 and Firefox versions 2 through 21 (all inclusive), yes, the old standard was implemented, **but** it was **vendor-prefixed**.
